package/MDAnalysis/auxiliary/base.py

```diff
@@ -323,7 +324,16 @@ def __init__(self, represent_ts_as='closest', auxname=None, cutoff=None,
             self.rewind()
 
     def copy(self):
-        raise NotImplementedError("Copy not implemented for AuxReader")
+        """Returns a deep copy of the AuxReader"""
+        orig_dict = pickle.dumps(self)
+        new_reader = pickle.loads(orig_dict)
+        return new_reader
+
+    def __getstate__(self):
```

Comment:
Wait @BFedder quick query does the pickling work without `__getstate__` and `__setstate__` defined here? IIRC pickle does have a default that is very similar to this that does not require the methods to be explicitly defined. 
See [here](https://docs.python.org/3/library/pickle.html#object.__getstate__)